pages/index.js

```diff
@@ -18,36 +21,129 @@ import {
   LatestBlockTXNIcon,
   SecondsToBlockIcon,
   TotalSupplyIcon,
-} from 'svgx'
+} from 'svgx';
 
+const LastBlockInfo = () => {
+  const cardWidth = useBreakpointValue({
+    base: '100%',
+    sm: 'calc(50% - 1rem)',
+    md: 'calc(33% - 1rem)',
+  });
+  const headBlock = useBlockHead();
 
-const useBlocks = () => ({
-  loaded: true,
-  data: new Array(14).fill({
-    sequence: 23940194,
-    size: 4096,
-    transactions_count: 1,
-    hash: '0000456456654564346345644564000045645665456434634564456445644564',
-    timestamp: '6/22/2021 8:09:45PM'
-  })
-})
+  const getValue = (field, transform = (value) => value) => {
+    return headBlock.loaded ? (
+      transform(headBlock.data[field])
+    ) : (
+      <span>&nbsp;</span>
+    );
+  };
```

Comment:
I think this might be clearer without using scoped variables. Also, is `headBlock.data[field]` guaranteed to exist?

```ts
const identity = (x) => x
const transformValue = (headBlock, field) => {
  return headBlock.loaded ? transform(headBlock.data[field]) : <span>&nbsp;</span>
}
// [...]
const getValue = transformValue(headBlock, field, identity)
```